batch_uploads_imageuploader.py

```diff
@@ -0,0 +1,147 @@
+
+def dicomUpload(filepath, isphantom, patientname):
+    pscid, candid, visit_label = patientname
+    filename = os.path.basename(filepath)
+    files = {'mriFile': (filename, open(filepath, "rb"), 'application/x-tar')}
+    payload = {
+        'IsPhantom': 'true' if isphantom == 'Y' else 'false',
+        'CandID': candid,
+        'PSCID': pscid,
+        'Visit': visit_label
+    }
+    response = requests.post(
+        url = args.baseurl + '/candidates/' + candid + '/' + visit_label + '/dicoms/',
+        headers = {'Authorization': 'Bearer %s' % jwtoken, 'LORIS-Overwrite': 'overwrite'},
+        verify = False,
+        data = payload,
+        files = files
+    )
+    responsebody = response.content.decode('ascii')
+    if (response.status_code != 200):
+        raise ValueError(responsebody)
+    
+    
+    data = json.loads(responsebody)
+    return data['mri_uploads'].pop()['mri_upload_id']
+
+def startProcess(filename, patientname, mri_upload_id):
+    pscid, candid, visit_label = patientname
+    payload = {
+        'ProcessType': 'mri_upload',
+        'MRIUploadID': mri_upload_id
+    }
+    response = requests.post(
+        url = args.baseurl + '/candidates/' + candid + '/' + visit_label + '/dicoms/' + filename + '/processes',
+        headers = {'Authorization': 'Bearer %s' % jwtoken},
+        verify = False,
+        data = payload
+    )
+    if (response.status_code != 202):
+        raise ValueError(responsebody)
+        
+    return json.loads(response.content.decode('ascii'))
+
+if __name__ == '__main__':
+    
+    credentials = {
+        'username': input('LORIS username: '), 
+        'password': getpass.getpass('password: ')
+    }
+    
+    jwtoken = json.loads(requests.post(
+        url = args.baseurl + '/login',
+        json = credentials,
+        verify = False
+    ).content.decode('ascii'))['token']
+    
+    for line in open(args.inputfile):
+        if (len(line) == 0):
+           continue 
+
+        params      = line.split(' ')
+        if (len(params) != 3):
+            print('Invalid line: ' + line)
+            sys.exit(1)
+        filepath    = params[0]
+        isphantom   = params[1]
+        patientname = params[2].strip()
+        
+        try:
+            open(filepath, 'rb')
+        except FileNotFoundError as err:
+            print('Skipping ' + filepath)
+            print(err.args)
+            continue
+        
+        print('Uploading ' + filepath)
+        try:
+            mri_upload_id = dicomUpload(filepath, isphantom, patientname.split('_',2))
+        except ValueError as err:
+            print('Upload failed')
+            print(err.args[0])
+            continue
+            
+        print('Upload succesfull')
+        print('mri_upload_id: ' + str(mri_upload_id))
+        
+        if (args.autostart):
+            try:
+                responsebody = startProcess(os.path.basename(filepath), patientname.split('_',2), mri_upload_id)
```

Comment:
what happens here if the PatientName is `PSCID_CandID_VisitLabel_some_other_string`?

If I understand correctly, it would split like:
PSCID = PSCID
CandID = CandID
VisitLabel = `VisitLabel_some_other_string` instead of `VisitLabel` which would fail when entering the `VisitLabel` in the URL of the API. We should support both scenarios (`PSCID_CandID_VisitLabel` and `PSCID_CandID_VisitLabel_some_other_string`)